### DISCUSSION TITLE
GD Library extension not available with this php installation

### DISCUSSION THREAD

**Discussion by Tapiwa-1** [NativePHP/laravel#163] [Q&A] [answered]
after installation of native php on to the existing project that supported File uploads. It keeps on showing "GD Library extension not available with this php installation" but on the browser files are being uploaded successfully
![Screenshot from 2023-08-09 17-36-46](https://github.com/NativePHP/laravel/assets/83808601/0a9ef27a-2bea-49d5-854e-ea669f6c293f)

**Comment by simonhamp** [comment, marked as answer]
[This has been raised as an Idea](https://github.com/orgs/NativePHP/discussions/281). Please vote for it there to let us know what we should prioritise.
